build.gradle

```diff
@@ -1,5 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext.kotlin_version = '1.0.0'
```

Comment:
Could you remove that please?

Reply:
Done. Sorry for that. Let's see how the CI ends up now.